pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -140,6 +140,9 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 }
 
 func (c *Controller) handleDriveUpdate(ctx context.Context, log *logrus.Entry, drive *drivecrd.Drive) (uint8, error) {
+	// handle drive labeled or not
+	if err := c.handleDriveLableUpdate(ctx, log, drive); err != nil {
```

Comment:
maybe better to deal with err

Reply:
This is intentional here now.  now label update should not block other functions.  and the err is also logged in function handleDriveLableUpdate.